### PR TITLE
[Method Reference followup Upgrade]Lps 182296 followup

### PR DESCRIPTION
**Pull request by slnn**
Hi @dantewang 

I update the folder name with a meaningful value and write the upgrade process, I just bump the micro version since it only modifies the data value, according to the Micro change examples in this document https://help.liferay.com/hc/en-us/articles/360017889792-Meaningful-Schema-Versioning#micro-change-examples

Lily

### CODE REVIEW

File: modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/util/UploadImageUtil.java [hunk 1]
```diff
@@ -34,7 +34,7 @@ public class UploadImageUtil {
 
 	public static final String TEMP_IMAGE_FILE_NAME = "tempImageFileName";
 
-	public static final String TEMP_IMAGE_FOLDER_NAME = "java.lang.Class";
+	public static final String TEMP_IMAGE_FOLDER_NAME = "tempImageFolderName";
```
Comment (dantewang) [inline modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/util/UploadImageUtil.java:37]: @slnn Please use `UploadImageUtil.class.getName()`. I believe this is what the original intent, but the author got it wrong.

Comment (slnn) [reply modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/util/UploadImageUtil.java]: Done

**Comment by slnn** [comment]
ci:test:relevant

**Comment by slnn** [comment]
ci:test:sf

**Comment by catarineaguiar** [comment]
ci:test:lima

**Review by dantewang — APPROVED** [review]

File: modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/upgrade/v1_0_1/DLFolderNameUpgradeProcess.java [hunk 2]
```diff
@@ -0,0 +1,40 @@
...
+				"update DLFolder set name = ? where name = ?")) {
+
+			preparedStatement.setString(
+				1, UploadImageUtil.TEMP_IMAGE_FOLDER_NAME);
+			preparedStatement.setString(2, "java.lang.Class");
+
+			preparedStatement.executeUpdate();
+		}
+	}
+
```
Comment (yuhai) [inline modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/upgrade/v1_0_1/DLFolderNameUpgradeProcess.java:39]: @slnn, when create one dl folder in UI, it also will create data (title column) in the assetentry table and reindex the dl. The current way won't do related changes. Please check if we need to invoke DLFolderLocalService api to do the change.

**Comment by catarineaguiar** [comment]
no regressions found.

**Comment by tinatian** [comment]
@slnn Please see @yuhai 's comment

**Comment by slnn** [comment]
Will update and resend.

**Comment by slnn** [comment]
Please see https://github.com/liferay-core-infra/liferay-portal/pull/4469

File: modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/upgrade/v1_0_1/DLFolderNameUpgradeProcess.java (hunk 2 above)
Comment (slnn) [reply modules/apps/image-uploader/image-uploader-web/src/main/java/com/liferay/image/uploader/web/internal/upgrade/v1_0_1/DLFolderNameUpgradeProcess.java:39]: Done

**Pull request closed without merging**